### PR TITLE
[RSDK-8626] Remove http2 builder customization in GrpcClient construction

### PR DESCRIPTION
**Pull request by acmorrow**

### CODE REVIEW

File: micro-rdk/src/common/grpc_client.rs [hunk 1]
```diff
@@ -238,10 +238,6 @@ impl GrpcClient {
     {
         let (http2_connection, conn) = {
             let client = hyper::client::conn::http2::Builder::new(executor.clone())
-                .initial_stream_window_size(4096)
-                .initial_connection_window_size(4096)
-                .max_concurrent_reset_streams(2)
-                .max_send_buf_size(4096)
```
Comment (mattjperez) [inline micro-rdk/src/common/grpc_client.rs:244]: not sure if we should leave a commented link to the defaults in the library, though updating that on every upgrade might be cumbersome.

**Review by mattjperez — APPROVED** [review]
LGTM

File: micro-rdk/src/common/grpc_client.rs (hunk 1 above)
Comment (acmorrow) [reply micro-rdk/src/common/grpc_client.rs:244]: We will never remember to do so and it will invariably become stale and therefore misleading

**Review by gvaradarajan — APPROVED** [review]
LGTM

**Review by npmenard — APPROVED** [review]
LGTM

**Pull request merged**